src/makeRobotAccountant.js

```diff
@@ -7,6 +7,29 @@
 
 function makeRobotAccountant() {
```

Comment:
<img width="651" alt="image" src="https://user-images.githubusercontent.com/38065883/173183768-093e9d68-adcc-4baa-8c30-3e6465d72aee.png">
<img width="883" alt="image" src="https://user-images.githubusercontent.com/38065883/173183774-4c5954ce-dc97-4d4b-b80c-5b6ae864ab59.png">

**_All even calls_**, not **_4th call_**